src/ImageSharp.Web/Memory/BufferDataPool.cs

```diff
@@ -6,9 +6,9 @@
 namespace SixLabors.ImageSharp.Web.Memory
 {
     /// <summary>
-    /// Provides a resource pool that enables reusing instances of value type arrays for image data <see cref="T:Byte[]"/>.
+    /// Provides a resource pool that enables reusing arrays for transporting encoded image data.
     /// </summary>
-    public class BufferDataPool
+    public class BufferDataPool : IBufferDataPool
```

Comment:
shouldn't this be replaced with `MemoryManager` from `ImageSharp` and flow the specified manager back into the configuration used during loading/resizing etc? 

This would require the  `Allocate`/`IBuffer<T>` instances being made public.

Reply:
However I wouldn't bother blocking the PR for this change, just something we aught to be aiming towards.

Reply:
Should’ve ran the tests before pushing. Poor form on my part. MemoryManager didn’t exist when I first wrote the web part but you’re right. I should refactor to now use it. Should be easy enough.